09_unique.rb

```diff
@@ -17,7 +17,37 @@
 # Check your solution by running the tests:
 # ruby tests/09_unique_test.rb
 #
+#
+#array style
+def unique (mylist)
+clean = []
+duplicate = []
```

Comment:
Looks awesome! You can ditch the duplicate array since we don't care about them. You could also remove the `else` condition.

Reply:
I didn't know what to do with it if it  was already in an array so I stashed it into duplicate.

Reply:
We don't need to do anything if we don't need it :) the `else` condition is optional